typespec-tests/src/main/java/com/resiliency/servicedriven/ResiliencyServiceDrivenClient.java

```diff
@@ -256,6 +270,29 @@ public void fromOneOptional(String parameter, String newParameter) {
         fromOneOptionalWithResponse(requestOptions).getValue();
     }
 
+    /**
+     * Tests that we can grow up an operation from accepting one optional parameter to accepting two optional
+     * parameters.
+     *
+     * @param parameter I am an optional parameter.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void fromOneOptional(String parameter) {
+        // Generated convenience method for fromOneOptionalWithResponse
+        RequestOptions requestOptions = new RequestOptions();
+        if (parameter != null) {
+            requestOptions.addQueryParam("parameter", parameter, false);
+        }
+        fromOneOptionalWithResponse(requestOptions).getValue();
+    }
```

Comment:
v1 would have API
```java
fromOneOptional()
fromOneOptional(String parameter)
```

v2 would have API
```java
fromOneOptional()
fromOneOptional(String parameter, String newParameter)
```

Hence here for v2 to be backward compatible, it now has
```java
fromOneOptional()
fromOneOptional(String parameter)  // for v1
fromOneOptional(String parameter, String newParameter)
```